ReactNativeClient/lib/joplin-renderer/MdToHtml.js

```diff
@@ -2,6 +2,13 @@ const MarkdownIt = require('markdown-it');
 const md5 = require('md5');
 const noteStyle = require('./noteStyle');
 const { fileExtension } = require('./pathUtils');
+
+const optionsCopy = {
+	failText: 'copy fail', // 'copy fail' | copy-fail text
+	successText: 'copied to clipboard!', // 'copy success' | copy-success text
+	successTextDelay: '2000', // 2000 | successText show time [ms]
+};
```

Comment:
This structure appears in two locations, but you don't even need it. Please put the text and delay directly where it's needed.